atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -0,0 +1,280 @@
+        if koji_task_id is not None:
+            self.log.info("build configuration created by Koji Task ID %s",
+                          koji_task_id)
+            try:
+                extra['container_koji_task_id'] = int(koji_task_id)
+            except ValueError:
+                self.log.error("invalid task ID %r", koji_task_id, exc_info=1)
+
+        fs_result = self.workflow.prebuild_results.get(AddFilesystemPlugin.key)
+        if fs_result is not None:
+            try:
+                fs_task_id = fs_result['filesystem-koji-task-id']
+            except KeyError:
+                self.log.error("%s: expected filesystem-koji-task-id in result",
+                               AddFilesystemPlugin.key)
+            else:
+                try:
+                    task_id = int(fs_task_id)
+                except ValueError:
+                    self.log.error("invalid task ID %r", fs_task_id, exc_info=1)
+                else:
+                    extra['filesystem_koji_task_id'] = task_id
+
+        help_result = self.workflow.prebuild_results.get(AddHelpPlugin.key)
+        if isinstance(help_result, dict) and 'help_file' in help_result and 'status' in help_result:
+            if help_result['status'] == AddHelpPlugin.NO_HELP_FILE_FOUND:
+                extra['image']['help'] = None
+            elif help_result['status'] == AddHelpPlugin.HELP_GENERATED:
+                extra['image']['help'] = help_result['help_file']
+            else:
+                self.log.error("Unknown result from add_help plugin: %s", help_result)
+
+        build = {
+            'name': component,
+            'version': version,
+            'release': release,
+            'source': "{0}#{1}".format(source.uri, source.commit_id),
+            'start_time': start_time,
+            'end_time': int(time.time()),
+            'extra': extra,
+        }
+
+        if self.metadata_only:
+            build['metadata_only'] = True
+
+        return build
+
+    def combine_metadata_fragments(self):
+        try:
+            metadata = get_build_json()["metadata"]
+            self.build_id = metadata["name"]
+        except KeyError:
+            self.log.error("No build metadata")
+            raise
+
+        metadata_version = 0
+
+        build = self.get_build(metadata)
+        """
+        # get all the platform names
+        build_result = self.workflow.build_result
+        annotations = build_result.annotations
+        worker_builds = annotations['worker-builds']
+        platforms = worker_builds.items()
+        """
+
+        metadatas = self.workflow.postbuild_results.get(FetchWorkerMetadataPlugin.key)
+        buildroot = self.get_buildroot(metadatas)
+        output = self.get_output(metadatas)
+
+        koji_metadata = {
+            'metadata_version': metadata_version,
+            'build': build,
+            'buildroots': [buildroot],
+            'output': [output],
```

Comment:
If `metadatas['output'] is a list: ['one', 'two', 'three'], and you wrap that in brackets: [['one', 'two', 'three']], you now have a list of one item. That item in itself is another list of multiple items. Don't think you want the surrounding brackets.